conda_lock/conda_lock.py

```diff
@@ -51,6 +51,9 @@
 # Captures the domain in the second group.
 DOMAIN_PATTERN = re.compile(r"^(https?:\/\/)?([^\/]+)(.*)")
 
+# Captures the platform in the first group.
+PLATFORM_PATTERN = re.compile(r"^# platform: (.*)$")
```

Comment:
More regex. 🙈 

If in the future the lockfile contains more metadata, maybe it would make sense to change the format to something that is easier to parse (e.g. `yaml`).

Reply:
technically this is yaml, if we strip the leading `"# "`

Reply:
true, i didn't think about it this way, lol.